packages/web-lib/.eslintrc.cjs

```diff
@@ -6,11 +6,7 @@ module.exports = {
 	},
 	'extends': [
 		'eslint:recommended',
-		'plugin:import/recommended',
-		'plugin:react/recommended',
 		'plugin:react-hooks/recommended',
-		'plugin:tailwindcss/recommended',
-		'plugin:@typescript-eslint/recommended',
```

Comment:
This was already done before here [10a447a9c70f798b6a266b45beb7b64f1972de2b](https://github.com/yearn/web-lib/commit/10a447a9c70f798b6a266b45beb7b64f1972de2b#diff-1b6ff05ea532e5adb70e2d18a489904f9dca6ab1ca85382970ea67b6b9987776L9) but somehow it came back